packages/client/src/profile/Profile.ts

```diff
@@ -0,0 +1,35 @@
+import { GraphQLClient } from 'graphql-request';
+
+import { ProfileFragment } from '../graphql/fragments.generated';
+import { SingleProfileQueryRequest } from '../graphql/types.generated';
+import { getSdk, Sdk } from './graphql/queries.generated';
+
+export class Profile {
+  private readonly sdk: Sdk;
+
+  constructor(backendUrl: string) {
+    const client = new GraphQLClient(backendUrl);
+    this.sdk = getSdk(client);
+  }
+
+  async fetch(
+    request: SingleProfileQueryRequest,
+    observerId?: string,
+  ): Promise<ProfileFragment | null> {
+    const result = await this.sdk.Profile({
+      request,
+      observerId,
+    });
+
+    return result.data.result;
+  }
+
+  async fetchDefault(address: string, observerId?: string): Promise<ProfileFragment | null> {
```

Comment:
The reason this edge exist is due to the fact Wallets-follow-Profiles in Lens Protocol v1.

It's very little publicized and underutilized.

In Lens Profile v2 Profiles will follow Profiles, I think we should omit it already from the SDK.

Reply:
so as a dev, how do I get my own profile after connecting a wallet? which query to use?

Reply:
sounds good, I will do the same here and remove `defaultProfile` query from the scope of this sdk